content/en/graphing/_index.md

```diff
@@ -9,128 +9,27 @@ aliases:
-
-## Save
-
-Click **Done** to save your work and exit the editor. You can always come back to the editor to change the graph. If you make changes you don't want to save, click **Cancel**.
+{{< whatsnext desc="This section includes the following topics:" >}}
+    {{< nextlink href="/graphing/using_graphs/" >}}<u>Using graphs</u>: Whether using metrics, monitors, dashboards, or notebooks, all graphs have basic functionality in common - start here for best practices for setting up graphs.{{< /nextlink >}}
+    {{< nextlink href="/graphing/dashboards" >}}<u>Dashboards</u>: A dashboard is Datadogs tool that visually tracks, analyzes, and displays key performance metrics to monitor the health of your infrastructure. Learn more about using Screenboards, Timeboards, optimizing those dashboards with template variables, and sharing your graphs to people outside of your Datadog account.{{< /nextlink >}}
+    {{< nextlink href="/graphing/metrics/" >}}<u>Metrics</u>: When you connect an integration to Datadog, the integration sends metrics about your system back to Datadog. Once Datadog receives these metrics, you can build graphs to explore the data. The Metrics Explorerlets you search for specific metrics and then create graphs based on your search parameters, and the Metrics Summary shows all of the metrics coming into Datadog.{{< /nextlink >}}
+    {{< nextlink href="/graphing/notebooks" >}}<u>Notebooks</u>: Notebooks combine graphs and text in a linear, cell-based format. They’re designed to help you explore and share stories with your data.{{< /nextlink >}}
+    {{< nextlink href="/graphing/event_stream/" >}}<u>Event Stream</u>: The event stream is a display of the most recent events in your infrastructure.{{< /nextlink >}}
+    {{< nextlink href="/graphing/infrastructure/" >}}<u>Infrastructure</u>: The Infrastructure list page shows all hosts monitored by your Datadog application. Host Maps visualize hosts together on one screen, with metrics made comprehensible via color and shape, and the container map and live containers shows you the health of your containers. Process Monitoring allows for real-time visibility of the most granular elements in a deployment. Cloud functions are a service that runs your code in response to events and automatically manage the underlying compute resources for you.{{< /nextlink >}}
+    {{< nextlink href="/graphing/functions/" >}}<u>From the query to the graph</u>: This sections goes through several topics to give you advanced graphing functions: Algorithms, Arithmetic, Count, Interpolation, Rank, Rate, Regression, Rollup, Smoothing, and Timeshift.{{< /nextlink >}}
+    {{< nextlink href="/graphing/graphing_json/" >}}<u>Graphing with JSON</u>: Use the API to create graphs.{{< /nextlink >}}
+    {{< nextlink href="/graphing/widgets/" >}}<u>Widgets</u>: Find all of the types of monitors you can create with Datadog.{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="/graphing/widgets/" >}}<u>Widgets</u>: Find all of the types of visualizations you can create with Datadog.{{< /nextlink >}}
```
Since some of these aren't really monitors... "visualizations" probably isn't the best word for them, either, but I guess it's not inaccurate since they are indeed visual? They're... widgets